### PR TITLE
Leaderboard: setup project

### PR DESCRIPTION
**Pull request by DannieMo**
### Task Completed
1. Create an npm project with webpack.
2. Write plain HTML markup with minimum styling

### CODE REVIEW

**Review by MahmoudBakr23 — COMMENTED** [review]
### Hi @DannieMo 😃, 

As much as I appreciate your hard work on this project, I'm afraid that I can't perform a review on this pull request.

**Invalid Submission**

- Please update your README file according to this [template](https://github.com/microverseinc/readme-template). 🙏 

_Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready._ 🙏

Cheers and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

<hr>

_Invalid Code Review Request does not count into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,2 +1,137 @@
...
+LeaderboardList App
+</b></h1>
+<p>
+In this activity I will set up a JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules. I will develop a first working version of the app following a wireframe, but without styling - just functionality. In the following activities (milestone 2 and 3), I will consume the Leaderboard API using JavaScript async and await and add some styling. </p> <hr>
+<p>
+The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service. </p>
+<br/>
+
+</div>
+
```
Comment (julie-ify) [inline README.md]: - [ ] Based on the template this project description should come after the table of contents under the "About the project" section. Kindly move this section to its proper place.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @DannieMo,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — APPROVED** [review]
Hi @DannieMo , 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**